### PR TITLE
login

### PR DESCRIPTION
**Pull request by aryan49sm**
completed login page

### CODE REVIEW

File: login.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+
```
Comment (the-special-character) [inline login.html]: Please add h1 with Login page title

**Review by the-special-character — CHANGES_REQUESTED** [review]
please make change according to comments

File: login.html (hunk 1 above)
Comment (aryan49sm) [reply login.html]: done sir

Comment (the-special-character) [reply login.html]: welldone

**Pull request merged**